src/components/DiagnosticsBody.vue

```diff
@@ -186,9 +186,12 @@ export default Vue.extend({
         <tr>
           <!--We want an empty data cell so description will align with name-->
           <td></td>
-          <td class="sub-row">
+          <td v-if="row.fixes.length > 0" class="sub-row">
             {{ row.fixes.description }}
           </td>
+          <td v-else>
+            (No fixes available)
+          </td>
```

Comment:
We might want to consider if conditionally rendering the sub-row toggle might be a better approach. I can imagine it being frustrating to expand a row only to find a "Nothing here" message when the same thing can be communicated by simple removing the button.

Reply:
I think we will discuss this in the review next week, but so far I think the collapsing functionality is not needed at all; the fixes should always be visible. And if there are no fixes, there is no need to add "No fixes"; that should be obvious by their absence. 😄

Reply:
I tried briefly to hide the toggle, but it didn't look easy so I opted for making a PR and improving later.

I suspect (once we have many diagnostics) that we may want to collapse by default, but that really needs a screen with many entries to tell for sure.